src/core/qgsmaplayer.cpp

```diff
@@ -3253,7 +3253,29 @@ QString QgsMapLayer::generalHtmlMetadata() const
         continue;
 
       const QVariant propValue = customProperty( key );
-      metadata += QStringLiteral( "<tr><td class=\"highlight\">%1</td><td>%2</td></tr>" ).arg( key.toHtmlEscaped(), propValue.toString().toHtmlEscaped() );
+      QString stringValue;
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
+      const QMetaType::Type propType = QgsVariantUtils::variantTypeToMetaType( propValue.type() );
+#else
+      const QMetaType::Type propType = propValue.metaType();
+#endif
+      if ( propType == QMetaType::QVariantList || propType == QMetaType::QStringList )
```

Comment:
```suggestion
      if ( propValue.type() == QVariant::List || propValue.type() == QVariant::StringList )
```

Reply:
Thanks a lot.

Btw. how do we consider the Qt-version situation? Are such version-checks not appropriated generally or just in this fix?